### ISSUE TITLE
Wabbajack Installation fail.  Mega upload file missing.

### ISSUE THREAD

**Issue by S-w-y-l-e-r** [TitansBane/Elysium#197]
https://mega.nz/file/UFtWmKgA#uFqxIunpV7rSgBuyUEQZIlCfqdzcazdDBnjaI3UGXHs

That file is no longer available.  Don't know for what mod.

**Comment by focused** [comment]
It's  "Elysium - Core Files .7z", I downloaded it earlier, put it inside WJ downloads, but WJ still fails with an error

**Comment by In2darabbithole** [comment]
I've got the same problem here.

**Comment by Mogor** [comment]
Same problem, megaupload file. 'no longer available', sorry!

**Comment by focused** [comment]
Any thoughts on when will it be solved? Can we manually download it somehow?

**Comment by michaelb1992** [comment]
hello im having the same problem also, i guess we will have to wait until the mod author releases an update :(

**Comment by Drewbies** [comment]
I'm having this problem as well.

**Comment by solemnhusk** [comment]
The issue is that the old download link Elysium - Core Files .7z is unavailable and has been reuploaded with a new link; however, wabbajack has not updated the modlist to the new link. When it tries to check the old link, it will fail, even if you have the file already downloaded.

TL;DR: The issue will be fixed once the wabbajack modlist is updated to the new mega link.

**Comment by focused** [comment]
The issue has been solved, PR should be closed

**Comment by michaelb1992** [comment]
> @S-w-y-l-e-r The issue has been solved, PR should be closed

what are you smoking bro its not fixed

**Comment by focused** [comment]
@michaelb1992 Go to WJ and select Elysium modlist from gallery, it must download the updated list version, once it finished click on play button to reinstall, it worked perfectly for me

**Comment by michaelb1992** [comment]
> @michaelb1992 Go to WJ and select Elysium modlist from gallery, it must download the updated list version, once it finished click on play button to reinstall, it worked perfectly for me

ok thank you very much i just tried downloaded it from local disk that's why, so i went to the browse mod list instead and updated elysium from there ,i cant wait to play :)

**Comment by focused** [comment]
The issue is being discussed here https://discord.com/channels/552625087438782474/882031813676126298
The last update happened about an hour ago

**Comment by michaelb1992** [comment]
> The issue is being discussed here https://discord.com/channels/552625087438782474/882031813676126298 The last update happened about an hour ago

thanks for the link much appreciated :)
